sct.py

```diff
@@ -1594,7 +1595,23 @@ def generate_parallel_timelines_report(logdir: str | None, test_id: str | None)
     pt_report_generator.generate_full_report()
 
 
-cli.add_command(configure_jenkins_builders)
+@cli.command("configure-jenkins-builders", help="Configure all required jenkins builders for SCT")
+@cloud_provider_option
+@click.option("-r", "--regions", type=CloudRegion(), default=[], help="Cloud regions", multiple=True)
```

Comment:
Are you sure that exactly instance of the `CloudRegion` class must be used here?
Maybe it must be exactly class itself?

Reply:
yes I'm sure, this is how this class is being used